### PR TITLE
add core notebooks as cards to example gallery

### LINKED ISSUE #379: add notebooks in `learn/core_notebooks` to the main examples gallery
Following quick discussion here https://github.com/pymc-devs/pymc-examples/issues/83#issuecomment-1159736382 we'll add the notebooks of core features (listed here https://www.pymc.io/projects/docs/en/stable/learn/core_notebooks/index.html) to the main set of examples here https://www.pymc.io/projects/examples/en/latest/gallery.html.

Add as the first section in the gallery

Core notebook files are located here https://github.com/pymc-devs/pymc/tree/main/docs/source/learn/core_notebooks

### PR DESCRIPTION
**Pull request by OriolAbril**
closes #379

### CODE REVIEW

**Comment by drbenvincent** [comment]
Looks good. Although just a few thoughts:
* Maybe the gallery view (icon size/alignment etc) can be the same for the core and regular examples. The gallery layout for the core notebooks seems better in that the images are bigger and they take up the full width of the page. That then also allows more space for text.
* Do we really need the note? On the one hand it might answer some questions if someone can't see the core notebooks in the tags etc. But on the other hand, I'm not totally convinced this is really useful information for the majority of people and it kind of creates a visual barrier to getting to the funky gallery images

**Comment by OriolAbril** [comment]
> Maybe the gallery view (icon size/alignment etc) can be the same for the core and regular examples. The gallery layout for the core notebooks seems better in that the images are bigger and they take up the full width of the page. That then also allows more space for text.

that would be good but I don't know how to do it. The differences are because the core notebooks use grids and cards provided by sphinx-design whereas the regular ones use the style provided by sphinx gallery.

> Do we really need the note? On the one hand it might answer some questions if someone can't see the core notebooks in the tags etc. But on the other hand, I'm not totally convinced this is really useful information for the majority of people and it kind of creates a visual barrier to getting to the funky gallery images

would moving the note below the core notebooks grid solve or partially solve this?

**Comment by drbenvincent** [comment]
I think it would improve things. Maybe good to get someone else's view on it as well to see if we need it at all.

Just a random thought... could we get more consistency by placing all the notebooks in pymc-examples and just pointing to them from the main docs?

**Comment by OriolAbril** [comment]
> Just a random thought... could we get more consistency by placing all the notebooks in pymc-examples and just pointing to them from the main docs?

We decided to have a handful of notebooks in the pymc repo to make sure there are always some notebooks that are up to date with the latest release. pymc-examples is too big of a collection and with too little contributions and reviews available to keep the notebooks up to date.

From what I have seen, even with some nb update pushes around pymc releases and the like, at least ~50% of the notebooks are consistently 2+ years old.

File: examples/gallery.rst [hunk 1]
```diff
@@ -6,6 +6,55 @@ PyMC Example Gallery
 
    object_index/index
 
+Core notebooks
+--------------
+
+.. note::
```
Comment (twiecki) [inline examples/gallery.rst:12]: Not sure this info is all that relevant to a user.

**Review by drbenvincent — APPROVED** [review]
Looks good. Would be great to figure out how to get the same kind of grid formatting for all the notebooks in the future.

**Review by twiecki — APPROVED** [review]

**Pull request merged**